lesson05/assignment/css/style.scss

```diff
@@ -1,6 +1,249 @@
 // Import partials here
-@import 'base.normalize';
-@import 'settings.variables';
-@import 'settings.responsive';
-@import 'components.content';
+@import "base.normalize";
+@import "settings.variables";
+@import "settings.responsive";
+@import "components.content";
 
+* {
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+$primary-color: #007193;
+$secondary-color: #b98077;
+$tertiary-color: #768a96;
+$quatenary-color: #ebe8e2;
+
+$min-width: 200px;
+
+@mixin transform($value) {
+  -webkit-transform: ($value);
+  -moz-transform: ($value);
+  -ms-transform: ($value);
+  transform: ($value);
+}
+@mixin transition($value) {
+  -webkit-transition: ($value);
+  -moz-transition: ($value);
+  -ms-transition: ($value);
+  transition: ($value);
+}
+
+%testing {
+  border: 5px solid blue;
+}
+
+%testing1 {
+  border: 5px solid red;
+}
+
+/* Page layout */
+//define grid area
+main {
+  nav {
+    grid-area: nav;
+  }
+  .side__navigation {
+    grid-area: side;
+  }
+  .main__description {
+    grid-area: content;
+  }
+}
+
+//grid template layout
+main {
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  grid-template-areas:
```

Comment:
you will also want to include your header and footer in your CSS grid. Otherwise on different screen sizes, your header and footer don't seem to keep up.

Reply:
I tried to add the header and footer as part of the grid before, but it skew up the layout of the main section (creating a blank space to the right of it that cannot be detected using the inspector).  I am not sure how to fix that - when you get a chance can you help me take a look?  Thank you!